app/controllers/v0/facilities/va_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class V0::Facilities::VaController < FacilitiesController
+  before_action :validate_params, only: [:index]
+
+  # Index supports the following query parameters:
+  # @param bbox - Bounding box in form "xmin,ymin,xmax,ymax" in Lat/Long coordinates
+  # @param type - Optional facility type, values = all (default), health, benefits, cemetery
+  # @param services - Optional specialty services filter
+  def index
+    results = VAHealthFacility.query(bbox: params[:bbox], services: params[:services])
+    render json: results,
+           serializer: CollectionSerializer,
+           each_serializer: VAHealthFacilitySerializer
+  end
+
+  def show
+    results = VAHealthFacility.find_by_id(id: params[:id])
+    raise Common::Exceptions::RecordNotFound, params[:id] if results.nil?
+    render json: results, serializer: VAHealthFacilitySerializer
+  end
+
+  private
+
+  def validate_params
+    bbp = params[:bbox].to_s.split(',')
```

Comment:
In this case where this is not data coming from a form but a search query, do you think the consistency argument still applies here?

I could see it be kind of confusing for client to have to construct URLs in this form instead of a comma separated string?